using-valtimo/plugin/plugins.md

```diff
@@ -7,7 +7,6 @@ This sections contains information on how plugins can be created, configured and
 * [Configure Plugin](configure-plugin.md)
 * [Creating a process link](create-process-link.md)
 * [Deleting a process link](delete-process-link.md)
-* [SmartDocuments plugin](smartdocuments/configure-plugin.md)
```

Comment:
This link should not be deleted. We do have documentation. The link above should reference this page:
https://docs.valtimo.nl/using-valtimo/plugins/configure-smartdocuments-plugin

Reply:
I can see it working as well. Closing this PR.